outputs.tf

```
@@ -18,6 +18,11 @@ output "endpoint" {
   description = "The connection endpoint"
 }
 
+output "name" {
+  value       = aws_redshift_cluster.default.cluster_identifier
+  description = "The name the cluster can be identified by"
+}
```

Why not call this `cluster_identifier`? E.g.

```suggestion
output "cluster_identifier" {
  value       = aws_redshift_cluster.default.cluster_identifier
  description = "The cluster identifier"
}
```